test/unit/gateways/merchant_warrior_test.rb

```diff
@@ -90,11 +90,11 @@ def test_scrub_name
     end.respond_with(successful_purchase_response)
   end
 
-  def test_description_truncated
```

Comment:
The goal of this test was to verify that we're actually truncating a really long value.